core-bundle/src/Search/Backend/Provider/TableDataContainerProvider.php

```diff
@@ -202,7 +202,13 @@ private function findDocuments(string $table, ReindexConfig $reindexConfig): \Ge
 
         $searchableFields = array_filter(
             $fieldsConfig,
-            static fn (array $config): bool => isset($config['search']) && true === $config['search'],
+            static function (array $config): bool {
+                if (\array_key_exists('backendSearch', $config)) {
+                    return $config['backendSearch'];
+                }
+
+                return \array_key_exists('search', $config) && true === $config['search'];
```

Comment:
Shouldn't this just be…
```suggestion
                return $config['search'] ?? false;
```

… or that (but then also for `backendSearch`)?
```suggestion
                return (bool)$config['search'] ?? false;
```

Reply:
Yeah, CS...whatever :)